index.js

```diff
@@ -1,7 +1,11 @@
-import fallback from 'json-parse-even-better-errors';
 import {codeFrameColumns} from '@babel/code-frame';
 import indexToPosition from 'index-to-position';
 
+const getCodePoint = character => {
+	const codePoint = character.codePointAt(0).toString(16).toUpperCase();
+	return '0x' + (codePoint.length % 2 ? '0' : '') + codePoint;
```

Comment:
I prefer showing it as a Unicode escape: https://github.com/sindresorhus/unicode-escapes/blob/ceb0401f6914ca1291819eb2822c38dd6294b8fb/index.js#L13

Reply:
DONE.